testing/constraints-3.7.txt

```diff
@@ -9,7 +9,6 @@ googleapis-common-protos==1.56.2
 protobuf==3.19.5
 google-auth==2.14.1
 requests==2.18.0
-packaging==14.3
```

Comment:
why do we remove this?

Reply:
`packaging` is not a direct dependency. It doesn't need to be in `testing/constraints-3.7.txt`. IOW, tests pass regardless if it is installed or not